### DISCUSSION TITLE
How to handle complaints?

### DISCUSSION THREAD

**Discussion by plancked** [postalserver/postal#3228] [Development discussions]
Hi,

I have setup postal with mailwizz.
Everything works, except that complaints are not handled correctly. When hitting the "Mark as spam" button on yahoo or gmail, nothing happens.

It seems that postal does not include complaints on its webhooks?
https://forum.mailwizz.com/threads/abuse-fbl-with-postal.7799/#post-47792

Is there a workaround to this?

Thanks
Alex

**Comment by z3d3m0n** [comment]
great question .  would love to learn this one aswel

**Comment by willpower232** [comment]
I'm not sure that Postal receives any information about the message being rejected by the end recipient so does not have anything to forward?

**Comment by TR-SametOzen** [comment]
Hello,

I'm curious about the MailVizz structure. Any chance of showing me a demo?

**Comment by plancked** [comment]
You can view at https://www.mailwizz.com/

I tested this with Sendy as well. It does not work. Spam complaints for mails sent via Postal are not received.

I would like to hear from the developers about this?

Thanks!

**Comment by plancked** [comment]
Anyone?
